third_party/cpaembedded/embedded/kiro_executor.go

```
@@ -0,0 +1,514 @@
+			var input any = map[string]any{}
+			if strings.TrimSpace(event.ToolInput) != "" {
+				_ = json.Unmarshal([]byte(event.ToolInput), &input)
+			}
+			blocks = append(blocks, map[string]any{
+				"type": "tool_use", "id": event.ToolUseID, "name": event.ToolName, "input": input,
+			})
+		case kiroEventMetadata:
+			usage = kiroAnthropicUsage(event)
+			stopReason = "end_turn"
+		case kiroEventInvalidState, kiroEventException:
+			if msg := event.ErrorText(); msg != "" {
```

golangci-lint (2.13.2)

[error] SA9003: empty branch

(staticcheck)